Components/Toast/toast.css

```diff
@@ -0,0 +1,139 @@
+/* ///////baseline///// */
+
+.baseline {
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+  font-size: large;
+  background-color: rgb(54, 53, 53);
+  width: fit-content;
+  border-radius: 5px;
+  padding: 5px 10px;
+}
+
+.baseline-para {
+  color: whitesmoke;
+}
+
+.baseline-retry {
+  display: flex;
+  color: rgb(223, 93, 223);
+  align-items: center;
+  padding: 10px 10px 10px 10px;
```

Comment:
You can use shorthand property for padding here.

Reply:
will change it